src/lib_ccx/ocr.c

```diff
@@ -296,15 +298,6 @@ char* ocr_bitmap(void* arg, png_color *palette,png_byte *alpha, unsigned char* i
 				int x1, y1, x2, y2;
 				if (!TessPageIteratorBoundingBox((TessPageIterator *)ri, level, &x1, &y1, &x2, &y2))
 					continue;
-				// printf("word: '%s';  \tconf: %.2f; BoundingBox: %d,%d,%d,%d;",word, conf, x1, y1, x2, y2);
```

Comment:
Not sure we want to completely remove this. Seems like code that whoever is doing maintenance/debugging may want to enable from time to time.

Reply:
Fair point, but printf literally doesn't even work here. I'll change it to a mprint()